stubs/CoreGenericFunctions.phpstub

```diff
@@ -334,6 +334,22 @@ function fclose(&$stream) : bool
 {
 }
 
+/**
+ * @psalm-pure
+ * @template T as string
+ * @param T $string
+ * @return (T is non-empty-string ? non-empty-string : string)
+ */
+function sodium_bin2base64(string $string, int $id): string
+
+/**
+ * @psalm-pure
+ * @template T as string
```

Comment:
just a headsup from the review in phpstan. we don't need a template in phpstan for this case, see https://github.com/phpstan/phpstan-src/pull/1664

maybe thats also the case for psalm and this can be simplified

Reply:
yeah, same on Psalm, but stubs are transparent for users so it doesn't really matters